app/src/androidTest/java/ch/epfl/sdp/peakar/social/SocialActivityTest.java

```diff
@@ -60,8 +65,26 @@ public void TestMenuBarSocial(){
         MenuBarTestHelper.TestSelectedIconButton(R.id.menu_bar_social);
     }
 
+    /* Test that the top bar color is correct */
     @Test
     public void TestTopBarColor() {
         onView(ViewMatchers.withId(R.id.top_bar)).check(matches(withBackgroundColor(R.color.LightGrey)));
     }
+
+    /* Test that the switch is visible and has the correct text. */
+    @Test
+    public void TestSwitchNames() {
+        onView(ViewMatchers.withId(R.id.top_bar_switch)).check(matches(isDisplayed()));
+        onView(ViewMatchers.withId(R.id.top_bar_switch_text_left)).check(matches(withText("All")));
+        onView(ViewMatchers.withId(R.id.top_bar_switch_text_right)).check(matches(withText("Friends")));
```

Comment:
The strings _All_ and _Friends_ are hard coded. 
Maybe we could add two public static strings in the `SocialActivity` so we can easily change them and tests keep working?

Reply:
I am using strings xml to set them in the SocialActivity. Maybe I can take the strings from there instead of this?

Reply:
Yeah yeah, we can use the strings from the XML file then.